### PR TITLE
Put the logo on the docs page.

### PR DESCRIPTION
**Pull request by ueshin**
Screen Shot 2019-09-25 at 5 53 47 PM

### CODE REVIEW

**Comment by ueshin** [comment]
I'm not sure this is the best place, though. cc @gioa @HyukjinKwon @gatorsmile

File: docs/source/index.rst [hunk 1]
```diff
@@ -1,5 +1,8 @@
 .. Koalas' documentation master file
 
+.. image:: ../../Koalas-logo-nolabel.png
+    :width: 160px
```
Comment (HyukjinKwon) [inline docs/source/index.rst]: How about placing the logo like pyspark? (https://spark.apache.org/docs/latest/api/python/index.html)
I think it's also fine to just don't put the logo - seems pandas doesn't do that as well

Comment (ueshin) [reply docs/source/index.rst]: I got an advice from our designer.

Comment (HyukjinKwon) [reply docs/source/index.rst]: +1 for the current one

**Comment by ueshin** [comment]
Thanks @gioa for the great advice! merging.

**Pull request merged**